pythonx/UltiSnips/snippet_manager.py

```diff
@@ -144,6 +145,8 @@ def __init__(self, expand_trigger, forward_trigger, backward_trigger):
         self._should_update_textobjects = False
         self._should_reset_visual = False
 
+        self._storeManager = StoreManager()
```

Comment:
consistency: use `_store_manager` as everywhere.

Reply:
Oh yeah sorry, I corrected the indent but forgot about the case I will add some commits anyway